components/websubhub/hub_state_update.bal

```diff
@@ -18,19 +18,36 @@ import websubhub.common;
 import websubhub.config;
 import websubhub.connections as conn;
 
-import ballerina/http;
+// import ballerina/http;
 import ballerina/lang.value;
+import ballerina/log;
 import ballerina/websubhub;
 import ballerinax/kafka;
 
 function initializeHubState() returns error? {
-    http:Client stateSnapshotClient = check new (config:stateSnapshotEndpoint);
+    // http:Client stateSnapshot;
+    // common:HttpClientConfig? config = config:state.snapshot.config;
+    // if config is common:HttpClientConfig {
+    //     http:ClientConfiguration clientConfig = {
+    //         timeout: config.timeout,
+    //         retryConfig: config.'retry,
+    //         secureSocket: config.secureSocket
+    //     };
+    //     stateSnapshot = check new (config:state.snapshot.url, clientConfig);
+    // } else {
+    //     stateSnapshot = check new (config:state.snapshot.url);
+    // }
```

Comment:
Shall we remove the commented out code?

Reply:
Fixed with https://github.com/wso2/product-integrator-websubhub/pull/9/commits/f977bd46f65344239355dcfa996cf299192a4327